### ISSUE TITLE
'id_loss' doesnt reduce at all

### ISSUE THREAD

**Issue by RamkumarSakthivel** [dingwoai/FairMOT-BDD100K#4]
I am training hrnet-32 model from scratch on the BDD100k MOT dataset, but the id_loss doesn't reduce at all. It stays around 11 even after 10 epochs. Does anyone have any idea why?

**Comment by dingwoai** [comment]
reID is a classification task in this work. So it is difficult to reduce id_loss, especially with so many different IDs in a driving scene, which is the case in BDD100k MOT dataset.

**Comment by RamkumarSakthivel** [comment]
Got it, reID is a crossentropy loss. Thanks !
